interop/shared/src/main/scala/scalaz/zio/interop/future.scala

```diff
@@ -2,6 +2,7 @@ package scalaz.zio
 package interop
 
 import scala.concurrent.{ ExecutionContext, Future }
+import scala.language.implicitConversions
```

Comment:
I don't know much about your import policy. I put it here to be consistent with others imports although I prefer put it just above the `FutureToFutureOps` since it's the one needing it.